psq/queue_test.py

```diff
@@ -70,13 +81,23 @@ def testQueue(self):
 
             assert r.task_id == t.id
 
-    def testDequeue(self):
+    def test_dequeue(self):
         with patch('psq.queue.Queue._get_or_create_topic'):
             q = Queue(Mock())
             t = Task('1', sum, (1, 2), {'arg': 'c'})
             sub_mock = Mock()
             q._get_or_create_subscription = Mock(return_value=sub_mock)
 
+            # No messages
+            sub_mock.pull.return_value = []
+
+            tasks = q.dequeue()
+
+            assert sub_mock.pull.called
+            assert not tasks
```

Comment:
Nose can usually handle printing out just as nice an error message as `self.assertEqual`. So personally I avoid it, but I follow whatever the project uses.

In fact, I almost wrote all of these tests in 100% nose style without unittest (just functions).